### PR TITLE
Afk fixup

### PR DESCRIPTION
**Pull request by Cyberboss**
New configs: 
inactivity_period time until soft afk
afk_period time until hard afk and kick
kick_inactive changed to bool

### CODE REVIEW

**Comment by Davidj361** [comment]
Hardcoded values make my peepee hard

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -408,10 +410,12 @@
...
 					allow_Metadata = 1
+				if("inactivity_period")
+					inactivity_period = text2num(value)
+				if("afk_period")
+					afk_period = text2num(value) * 10 //documented as seconds in config.txt
 				if("kick_inactive")
-					if(value < 1)
-						value = INACTIVITY_KICK
-					kick_inactive = value
+					kick_inactive = text2num(value) * 10 // ^^^
```
Comment (Jordie0608) [inline code/controllers/configuration.dm]: This is meant to be the boolean.

File: code/controllers/configuration.dm [hunk 2]
```diff
@@ -408,10 +410,12 @@
 					usewhitelist = TRUE
 				if("allow_metadata")
 					allow_Metadata = 1
+				if("inactivity_period")
+					inactivity_period = text2num(value)
```
Comment (Jordie0608) [inline code/controllers/configuration.dm]: And this should be the `* 10`

File: config/config.txt [hunk 3]
```diff
@@ -81,7 +81,13 @@ LOG_LAW
 ## As an example of how this could be """useful""" look towards Poly (https://twitter.com/Poly_the_Parrot)
 # LOG_TWITTER
 
-## disconnect players who did nothing during 10 minutes
+## period of time in seconds for players to be considered inactive
+# INACTIVITY_PERIOD 300
```
Comment (Jordie0608) [inline config/config.txt]: You've mixed around this and `AFK_PERIOD`'s values.

Comment (Cyberboss) [inline config/config.txt]: fixed in config

File: code/controllers/subsystem/server_maint.dm [hunk 4]
```diff
@@ -10,12 +10,13 @@ SUBSYSTEM_DEF(server_maint)
...
+	if(config.kick_inactive)
 		for(var/client/C in GLOB.clients)
-			if(C.is_afk(INACTIVITY_KICK))
-				if(!istype(C.mob, /mob/dead))
+			if(C.is_afk(config.afk_period))
+				var/cmob = C.mob
+				if(!(istype(cmob, /mob/dead/observer) || (istype(cmob, /mob/dead) && C.holder)))
 					log_access("AFK: [key_name(C)]")
-					to_chat(C, "<span class='danger'>You have been inactive for more than 10 minutes and have been disconnected.</span>")
+					to_chat(C, "<span class='danger'>You have been inactive for more than [config.afk_period / 60] minutes and have been disconnected.</span>")
```
Comment (duncathan) [inline code/controllers/subsystem/server_maint.dm]: shouldn't this be /600 since it's stored as ds?

**Review by duncathan — CHANGES_REQUESTED** [review]

**Pull request merged**